web/res/js/donationForm.js

```diff
@@ -25,7 +25,7 @@ $( function () {
 			} ),
 			WMDE.Components.createRadioComponent( store, $( '.address-type-select' ), 'addressType' ),
 			WMDE.Components.createRadioComponent( store, $( '.salutation' ), 'salutation' ),
-			WMDE.Components.createRadioComponent( store, $( '.personal-title' ), 'title' ),
+			WMDE.Components.createSelectMenuComponent( store, $( '.personal-title' ), 'title' ),
```

Comment:
What was the actual issue? I suppose `createRadioComponent` is not broken since it is still used above. Did this mismatch with code elsewhere?